Gemfile.jruby-3.1.lock.release

```diff
@@ -868,6 +866,7 @@ GEM
 
 PLATFORMS
   java
+  universal-java-11
```

Comment:
**Note for reviewers**

I don't know why this dependency was kicked in, maybe could be removed

Reply:
+++ to remove, we need to either find a way to avoid this or to auto yank it before pushing the commit/pr in the github action.